.github/workflows/linters.yml

```diff
@@ -4,41 +4,23 @@ on:
   - pull_request
 
 jobs:
-  ruff:
+  pre-commit-hooks:
     runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: chartboost/ruff-action@v1
+    strategy:
+      matrix:
+        hook_id:
```

Comment:
My only concern about having the hooks listed in the GH workflow is that we need to keep them in 2 places so they might get off sync.

Reply:
Yes, that's right... but we want to have separated CI jobs for each hook, as discussed [in Slack](https://redhat-internal.slack.com/archives/C01A4DTTZN1/p1736753937505169?thread_ts=1736528153.367079&cid=C01A4DTTZN1)